content/guides/getstarted/how-to-create-a-token.md

```diff
@@ -0,0 +1,327 @@
+- [Solana Explorer](https://explorer.solana.com/address/niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun?cluster=devnet)
+  etc.
+
+Make sure your explorer is set to devnet if you are working on dev net and
+replace the address to your mint address starting with "nic".
+
+## Update meta data
+
+At any point as long as you do not set the update authority to null you will
+still be able to do changes the meta data of this token.
+
+Let's say you want for example update the uri field of the token. You can do
+this by running the following command:
+
+```bash
+spl-token update-metadata niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun \
+uri https://bafybeidfm65jzvz4zeesxp6ybinkitvpd27klk6yspstrtw5fuy5w27lkq.ipfs.w3s.link/metadata.json
+```
+
+You can also add additional meta data to your token as a key-value pair like
+this:
+
+```bash
+spl-token update-metadata niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun niceness 100%
+```
+
+This is more useful for NFTs but you can also use it for fungible tokens.
+
+## Mint tokens
+
+Now that the token is completely set up we can mint some tokens. Let's mint 100.
+
+First we need to create a token account to hold the tokens for our token mint:
+
+```bash
+spl-token create-account niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun
+```
+
+This will create a new token account for the account that is currently set in
+the solana config. You can also specify a different account by adding the
+address at the end of the command.
+
+And now we can finally mint some tokens into that token account:
+
+```bash
+spl-token mint niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun 100
+```
+
+Now you can also send the token to your friends like so for example:
+
+```bash
+spl-token transfer niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun 10 \
+friendAddress --fund-recipient
+```
+
+"--fund-recipient" means that you will pay to create the token account for your
+friend.
+
+## Creating token markets
+
+Now that your token is created you probably want to create a market and a pool
+for it and have it listed on exchanges. There are many ways to create a market
+for your token. Here are some possible solutions:
+
+- [Radium](https://docs.raydium.io/raydium/pool-creation/creating-a-standard-amm-pool)
+- [Orca](https://docs.orca.so/orca-for-liquidity-providers/classic-community-listing/creating-a-pool-tutorial)
+- [Meteora](https://docs.meteora.ag/liquidity-primitives/dynamic-amm-pools/permissionless-dynamic-pools)
+- [Mango Markets](https://docs.mango.markets/)
+- [Jupiter](https://station.jup.ag/docs/get-your-token-onto-jup)
+
+If you have your own token market platform feel free to open a PR to add it
+here.
+
+## Community tools to create tokens
+
+Use these at your own risk. In general it is recommended to never give out any
+private keys and using external tools may expose you to security risks and give
+the tool creators control over your token. These make it easy to create token,
+but can also come with fees.
```

Comment:
```suggestion
Use these at your own risk. These tools may make it easy to create tokens, but may also expose you to security risks, add additional fees, and give the tool creators control over your token. 

A reminder to never give your private keys to other people.
```

Reply:
This was great, just tightening the wording a little bit to make the risks more obvious.